src/ownership-and-deconstruction.rst

```diff
@@ -637,6 +641,10 @@ An :dt:`extending expression` is either
   The :t:`tail expression` of a :t:`block expression` that is an
   :t:`extending expression`.
 
+* :dp:`fls_eeaJtK4w5gVK`
+  The :t:`call expression` that is the :t:`extending expression`
+  where the :t:`call operand` resolves to a :t:`tuple struct` or a :t:`tuple enum variant`.
```

Comment:
This item has to recurse downward.  E.g., the one above, "the tail expression of a block expression that is an extending expression".  It's working its way down.  Or, e.g., consider the Reference text:

> The arguments to an extending tuple struct or tuple enum variant constructor expression.

The text in this PR isn't correct yet because it's not referring to the arguments anywhere here.

Also, you probably want to line wrap, here and elsewhere in the PR, to 80 to be consistent with the other text, at least until an FLS team makes a decision about going a different direction with this.